data/api/client-server/relations.yaml

```diff
@@ -0,0 +1,170 @@
+        required, they can be individually omitted to retrieve all matching events.
+        When omitting the path parameters, a trailing slash must not be included
+        (otherwise the search will be for an empty string).
+
+        When combining the use of `relType` and `eventType`, the server will return
+        child events which match *both* conditions rather than *either*.
+      operationId: getChildEventsForParentEvent
+      security:
+        - accessToken: []
+      parameters:
+        - in: path
+          type: string
+          name: roomId
+          description: The ID of the room the supposed parent event is in.
+          required: true
+          x-example: "!636q39766251:matrix.org"
+        - in: path
+          type: string
+          name: eventId
+          description: The supposed parent event ID.
+          required: true
+          x-example: "$asfDuShaf7Gafaw"
+        - in: path
+          type: string
+          name: relType
+          description: |-
+            The [relationship type](/client-server-api/#relationship-types) to search
+            for, if any. Exclude to find all child events which point to the supposed parent
+            event with any `rel_type`.
+          required: true # we can't say false, annoyingly
+          x-example: "org.example.my_relation"
+        - in: path
+          type: string
+          name: eventType
+          description: |-
+            The event type of child events to search for, if any. Exclude to find all
+            child events of any type which point to the supposed parent event.
+
+            Note that in encrypted rooms this will typically always be `m.room.encrypted`
+            regardless of the event type contained within the encrypted payload.
+          required: true # we can't say false, annoyingly
+          x-example: "m.room.message"
+        - in: query
+          type: string
+          name: from
+          description: |-
+            The pagination token to start returning results from. If not supplied, results
+            start at the earliest topological event known to the server.
+
+            Can be a `next_batch` or `prev_batch` token from a previous call, or an equivalent
+            token from [`/messages`](/client-server-api/#get_matrixclientv3roomsroomidmessages)
+            or [`/sync`](/client-server-api/#get_matrixclientv3sync) to limit results to the
+            events returned by that section of timeline.
+          required: false
+          x-example: "page2_token"
+        - in: query
+          type: string
+          name: to
+          description: |-
+            The pagination token to stop returning results at. If not supplied, results
+            continue up to `limit` or until there are no more events.
+
+            Like `from`, this can be a previous token from a prior call to this endpoint
+            or from `/messages` or `/sync` to limit to a section of timeline.
+          required: false
```

Comment:
the MSC doesn't even bother to define `to`. *throws hands up in despair*

Reply:
this appears to have been an MSC bug, ftr.